tools/pylib/boututils/moment_xyzt.py

```diff
@@ -30,8 +29,6 @@ def moment_xyzt( sig_xyzt, *args):#rms=None, dc=None, ac=None):
 #; -AC (DC subtracted out), i.e., a function of (x,y,z,t)
 #;-------------------------------------------------------------------
```

Comment:
It wasn't changed in this PR, but since we're touching the methods here...
This `moment_xyzt` seems to be mis-named - it's actually using `(t,x,y,z)` (I guess this is due to a port from IDL to Python). It would be nice to change the names and comments so that they reflect the actual `(t,x,y,z)` behaviour.

Reply:
Sorry, but I would rather remove them.
I think what the function does is to simply to put actually in a library ...

Reply:
I don't have strong feelings about it. Given what @ZedThree said in the BUG meeting today about getting rid of stuff in the old pylib, I'd vote for leaving as is for now (get this PR merged), and remove stuff from pylib in a dedicated PR.